zad1/main.rb

```diff
@@ -1,21 +1,23 @@
 def sort_letters(string)
-
+    print(string,'->',string.chars.sort.join,"\n")
 end
 
 def count_vowels(string)
-
+    print("\"",string,"\"",' zawiera: ',string.scan(/[aeiouy]/).count,' samoglosek',"\n")
+    #puts(string.scan(/[aeiouy]/).count)
 end
 
 def filter_odd(array)
-
+    #puts array.select &:odd?
+    print array, " -> ", array.select{|a| a.odd?},"\n"
```

Comment:
Tu mogłeś skorzystać z shorthand z prezentacji czyli:

```ruby
array.select(&:odd?)

Reply:
Ad. 1 Czyli gdybym to wyrażenie napisał  w formie np.: 
```
def count_vowels(string)`
    string.scan(/[aeiouy]/).count,
end
```

to to by wystarczyło?

Ad. 2. Jasna sprawa - dało radę to skrócić jeszcze bardziej, co widać w skomentowanej linii w moim kodzie powyżej. Przy którejś ze zmian kodu musiałem o tym zapomnieć :)

Reply:
1. Wystarczyłoby :) Żeby było jaśniej powinniśmy dać puts przy przykładach :(